netutils/config/parser.py

```diff
@@ -588,6 +589,68 @@ def build_config_relationship(self) -> t.List[ConfigLine]:
         self._update_same_line_children_configs()
         return self.config_lines
 
+    def _get_groups(self, pattern: str) -> t.Any:
+        """Groups children based on parent pattern."""
+        children_list = []
+        for line in self.config_lines:
+            for parent in line.parents:
+                if re.match(pattern, parent):
+                    children_list.append(line)
+        grouped_data = it.groupby(children_list, key=lambda x: x.parents)  # type: ignore
+        return grouped_data
+
+    def get_path(self, pattern: str) -> t.List[str]:
+        """Returns configuration part for specific pattern not including parents.
+
+        Args:
+            pattern (str): pattern that describes for parent.
+
+        Returns:
+            list: configuration under that parent pattern.
+
+        Example:
+        root_obj.get_path("router bgp 45000")
+        # Returns
+         address-family ipv4 unicast
+          neighbor 192.168.1.2 activate
+          network 172.17.1.0 mas
+        """
+        children = []
```

Comment:
I think method could be solved with `[a.config_line for a in config_tree.build_config_relationship() if a.parents and a.parents[0] == pattern]`

Reply:
Changed

Reply:
What you are suggesting does not return anything, it returns an empty list.